2020-06-02 23:19:55 [scrapy] DEBUG: Crawled (200) <GET https://www.yelp.com/> (referer: None)
2020-06-02 23:20:22 [scrapy] ERROR: Spider error processing <GET https://www.yelp.com/> (referer: None)
Traceback (most recent call last):
  File "f:\web scrapping\yelp\venv\lib\site-packages\scrapy\utils\defer.py", line 117, in iter_errback
    yield next(it)
  File "f:\web scrapping\yelp\venv\lib\site-packages\scrapy\utils\python.py", line 345, in __next__
    return next(self.data)
  File "f:\web scrapping\yelp\venv\lib\site-packages\scrapy\utils\python.py", line 345, in __next__
    return next(self.data)
  File "f:\web scrapping\yelp\venv\lib\site-packages\scrapy\core\spidermw.py", line 64, in _evaluate_iterable
    for r in iterable:
  File "f:\web scrapping\yelp\venv\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 29, in process_spider_output
    for x in result:
  File "f:\web scrapping\yelp\venv\lib\site-packages\scrapy\core\spidermw.py", line 64, in _evaluate_iterable
    for r in iterable:
  File "f:\web scrapping\yelp\venv\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 338, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "f:\web scrapping\yelp\venv\lib\site-packages\scrapy\core\spidermw.py", line 64, in _evaluate_iterable
    for r in iterable:
  File "f:\web scrapping\yelp\venv\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "f:\web scrapping\yelp\venv\lib\site-packages\scrapy\core\spidermw.py", line 64, in _evaluate_iterable
    for r in iterable:
  File "f:\web scrapping\yelp\venv\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "f:\web scrapping\yelp\venv\lib\site-packages\scrapy\core\spidermw.py", line 64, in _evaluate_iterable
    for r in iterable:
  File "F:\web scrapping\yelpsecondinput\yelpdetails\yelpdetails\spiders\yelpspider.py", line 59, in parse
    print(driver.current_url)
  File "f:\web scrapping\yelp\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 669, in current_url
    return self.execute(Command.GET_CURRENT_URL)['value']
  File "f:\web scrapping\yelp\venv\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "f:\web scrapping\yelp\venv\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: chrome not reachable
  (Session info: chrome=83.0.4103.61)

2020-06-02 23:20:34 [scrapy] INFO: Closing spider (finished)
2020-06-02 23:20:36 [scrapy] INFO: Dumping Scrapy stats:
{'downloader/response_bytes': 1047352,
 'downloader/response_count': 1,
 'downloader/response_status_count/200': 1,
 'elapsed_time_seconds': 43.902069,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2020, 6, 2, 17, 50, 34, 391905),
 'httpcache/firsthand': 1,
 'httpcache/store': 1,
 'log_count/DEBUG': 56,
 'log_count/ERROR': 1,
 'log_count/INFO': 13,
 'response_received_count': 1,
 'scheduler/dequeued': 1,
 'scheduler/dequeued/memory': 1,
 'scheduler/enqueued': 1,
 'scheduler/enqueued/memory': 1,
 'spider_exceptions/WebDriverException': 1,
 'start_time': datetime.datetime(2020, 6, 2, 17, 49, 50, 489836)}
2020-06-02 23:20:36 [scrapy] INFO: Spider closed (finished)
